vars/buildPlugin.groovy

```diff
@@ -66,6 +66,17 @@ def call(Map params = [:]) {
                             isMaven = fileExists('pom.xml')
                             incrementals = fileExists('.mvn/extensions.xml') &&
                                     readFile('.mvn/extensions.xml').contains('git-changelist-maven-extension')
+                            if (incrementals) { // Incrementals needs 'git status -s' to be empty at start of job
```

Comment:
Could we not just do this unconditionally?

Reply:
We certainly could, but I wanted to keep the impact of the change to only those specific cases where I know it is an issue.  I'm willing to change the pull request to do it in all cases if that will help with its approval.